src/vm/frames.h

```diff
@@ -2466,17 +2466,22 @@ class ComPrestubMethodFrame : public ComMethodFrame
 //------------------------------------------------------------------------
 class GCFrame
 {
+    void Init(Thread *pThread, OBJECTREF *pObjRefs, UINT numObjRefs, BOOL maybeInterior);
+    void Push(Thread *pThread);
```

Comment:
These do not need to be separate methods either. Folding them in may help a code quality a little bit - I am not sure whether the C/C++ compiler is smart enough to inline these.

Reply:
Done, the compiler was not inlining them even in Release builds